### PR TITLE
update install instructions

### PR DESCRIPTION
**Pull request by dwightguth**

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -5,11 +5,13 @@ do any installation steps not listed explicitly.
 
 We recommend using Linux or OSX on a computer with at least 1 GB of memory.
 
-On Ubuntu, if the K Framework has already been installed (from here:
-<https://github.com/kframework/k/releases/tag/latest>), the
-installation process for our C semantics can be summarized as:
+On Ubuntu, the installation process for our C semantics can be summarized as:
 ```
-$ cd ~
+$ git clone --depth=1 https://github.com:runtimeverification/k.git
```
Comment (chathhorn) [inline INSTALL.md]: github.com:runtimeverification => github.com/runtimeverification

File: INSTALL.md [hunk 2]
```diff
@@ -63,7 +70,17 @@ the development packages for libcrypt, zlib, and libxml2.
 $ export PATH=$PATH:~/k/bin
 ```
 
-### 4. Install OCaml.
+### 4. Install OCaml K backend.
+- This is a proprietary component used to compile execute programs written in K.
```
Comment (chathhorn) [inline INSTALL.md]: compile execute => compile and execute

**Comment by ellisonch** [comment]
>
> A completely free executable version of the C semantics is not available
> at this time.


Any idea how long this state will last?

-Chucky

**Comment by dwightguth** [comment]
Ask the UIUC K team. It lasts until they can make the C semantics work in
the java backend again which so far has not been a high priority.

**Comment by dwightguth** [comment]
@grosu please review

**Comment by grosu** [comment]
LGTM.

@ellisonch : there are two implementations of K now, both public.  One maintained by the RV company aiming at performance, and another by UIUC aiming at innovation and experimentation.  They are both behind in what regards the Java backend.  The RV version has an OCAML backend, which is proprietary (the secret sauce), but we do offer in binary.

**Pull request merged**